### PR TITLE
feature/update-title

### PR DESCRIPTION
**Pull request by aslam-cyb**
Updated title logic to make it mandatory. Updated how pages are linked, use title as reference instead of path.

### CODE REVIEW

**Comment by aslam-cyb** [comment]
image

**Comment by aslam-cyb** [comment]
image

File: designer/client/link-create.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ class LinkCreate extends React.Component {
     const { data } = this.props
     const copy = clone(data)
     const page = copy.pages.find(p => p.path === from)
-
+    console.log('page', from)
```
Comment (superafroman) [inline designer/client/link-create.js:19]: Remove debug logging

File: designer/client/page-create.js [hunk 2]
```diff
@@ -8,8 +8,9 @@ class PageCreate extends React.Component {
     e.preventDefault()
     const form = e.target
     const formData = new window.FormData(form)
-    const path = formData.get('path').trim()
     const { data } = this.props
+    const title = formData.get('title').trim()
+    const path = '/'.concat(title.replace(/[^a-zA-Z ]/g, '').replace(' ', '-')).toLowerCase()
```
Comment (superafroman) [inline designer/client/page-create.js:13]: This is fine, so just a note worth knowing - this will strip non-English characters, e.g. á.

**Pull request merged**